containers/airunner.Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM ubuntu:22.04
+
+RUN apt-get update && apt-get install -y python3-pip git python-is-python3 less
+# All this stuff is needed to be able to render the airunner GUI (it already comes with Ubuntu-desktop usually!)
+RUN apt-get update && apt-get install -y '^libxcb.*-dev' libgl1 libsm6 libxrender1 libxext6 libglib2.0-dev libgl1-mesa-dev libxkbcommon-dev libdbus-1-3 libx11-xcb-dev libglu1-mesa-dev libxrender-dev libxi-dev libxkbcommon-dev libxkbcommon-x11-dev
```

Comment:
I'm not sure if we need these. I'm going to avoid adding this to the dockerfile - if this is required for a 20.04 build i think there should be a separate docker image for 20.04 builds (right now the docker image is using ubutu:latest)